tests/integration-tests/tests/schedulers/test_slurm/test_fast_capacity_failover/overrides.sh

```diff
@@ -14,22 +14,54 @@ slurm_plugin_path=$(sudo find / -iname slurm_plugin -print0|grep -FzZ 'node_virt
 cat > $slurm_plugin_path/overrides.py << EOF
 from botocore.exceptions import ClientError
 import boto3
+import logging
 
+logger = logging.getLogger(__name__)
 
 def run_instances(region, boto3_config, **run_instances_kwargs):
     if "ice-compute-resource" in run_instances_kwargs.get("LaunchTemplate", {}).get("LaunchTemplateName"):
+        run_instances_kwargs['Placement']= {'Tenancy': 'host'}
```

Comment:
please add a comment about the purpose of adding this parameter.
also consider doing `run_instances_kwargs['Placement']['Tenancy']=` so to not override the whole Placement

Reply:
Ooops, I missed the comment. I'll add it.

As for doing `run_instances_kwargs['Placement']['Tenancy']=` this is applied only on the target CR that doesn't have a `Placement`. As such the previous code will fail and I considered overkill and out-of-scope creating a more generic way to override this value since we are not going to use it.

Reply:
Added the missing comment